playbooks/roles/mattermost/tasks/mattermost.yml

```diff
@@ -32,10 +32,14 @@
     group: mattermost
     force: no
 
+# This autolink configuration is NOT used UNLESS you are installing MM from scratch.
+# You must MANUALLY copy it from /var/lib/mattermost/autolink.json to the correct place
+# in /var/lib/mattermost/config.json and restart MM.
+# This is explained in autolink.json.j2 but repeated here to be helpful.
 - name: generate autolink configuration
   template:
     src: autolink.json.j2
-    dest: "{{ MATTERMOST_CONFIG_FILE|dirname }}/autolink.json"
+    dest: "{{ MATTERMOST_STATUS_DIR }}/autolink.json"
```

Comment:
@SSPJ, this wasn't needed when I did the upgrade and the auto-linking works okay. So is this needed?

Reply:
No, it's just reducing some code complexity: `MATTERMOST_CONFIG_FILE` is `"{{ MATTERMOST_STATUS_DIR }}/config.json"` and so `MATTERMOST_CONFIG_FILE|dirname` is just `MATTERMOST_STATUS_DIR`.